jest.config.json

```diff
@@ -0,0 +1,24 @@
+{
+    "verbose": true,
+    "collectCoverage": true,
+    "coverageReporters": ["lcov", "json"],
+    "collectCoverageFrom": [
+        "src/**/*.{js,jsx,ts,tsx}",
+        "!<rootDir>/node_modules/"
+    ],
+    "transform": {
+        "^.+\\.ts?$": "ts-jest"
+    },
+    "testRegex": "/tests/.*\\.test\\.ts$",
+    "moduleFileExtensions": [
+        "ts",
+        "tsx",
+        "js",
+        "jsx",
+        "json",
+        "node"
+    ],
+    "moduleNameMapper": {
+        "@weco/(.+)$": "<rootDir>packages/shared/$1/src"
```

Comment:
In the `auth0-client` the rootDir is overridden to be `./packages/auth0-client` resulting in this mapping being: `./packages/auth0-client/packages/shared/$1/src` which does not exist.

Reply:
Isn't `<rootDir>` here relating to the jest config, not TypeScript? It does seem to work. I'll check with the change.